Frameworks/UIKit/UIViewController.mm

```diff
@@ -1222,9 +1230,9 @@ - (void)dismissViewControllerAnimated:(BOOL)animated completion:(void (^)(void))
     [curController retain];
     [curController autorelease];
 
-    BOOL popoverPresented = [curController _popoverManagesPresentation];
+    BOOL realPopoverPresented = [[curController popoverPresentationController] _isManagingPresentation];
```

Comment:
This sounds great, @ehren.  Can you please file an issue/suggestion tracking the simplification to always show a popup (in addition to the rest of the issues you've been hitting)?  Thanks!

---
In reply to: [90317212](https://github.com/Microsoft/WinObjC/pull/1464#discussion_r90317212) [](ancestors = 90317212)